src/app/wizard/component.ts

```diff
@@ -106,50 +107,30 @@ export class WizardComponent implements OnInit, OnDestroy {
     this._wizard.reset();
   }
 
-  onCreateCluster(): void {
-    this.creating = true;
-    let createdCluster: Cluster;
+  getObservable(): Observable<Cluster> {
     const createCluster = this._getCreateClusterModel(this._clusterSpecService.cluster, this._nodeDataService.nodeData);
 
-    this._clusterService
+    return this._clusterService
       .create(this.project.id, createCluster)
-      .pipe(
-        tap(cluster => {
-          this._notificationService.success(`Created the ${cluster.name} cluster`);
-          createdCluster = cluster;
-        })
-      )
-      .pipe(switchMap(_ => this._clusterService.cluster(this.project.id, createdCluster.id)))
-      .pipe(
-        switchMap(_ => {
-          this.creating = false;
-
-          if (this._clusterSpecService.sshKeys.length > 0) {
-            return forkJoin(
-              this._clusterSpecService.sshKeys.map(key =>
-                this._clusterService.createSSHKey(this.project.id, createdCluster.id, key.id)
-              )
-            );
-          }
-
-          return of([]);
-        })
-      )
-      .pipe(takeUntil(this._unsubscribe))
-      .subscribe({
-        next: (keys: SSHKey[]) => {
-          this._router.navigate([`/projects/${this.project.id}/clusters/${createdCluster.id}`]);
-          keys.forEach(key =>
-            this._notificationService.success(
-              `Added the ${key.name} SSH key to the cluster ${createCluster.cluster.name}`
-            )
+      .pipe(switchMap(cluster => this._clusterService.cluster(this.project.id, cluster.id)))
+      .pipe(takeUntil(this._unsubscribe));
+  }
+
+  onNext(cluster: Cluster): void {
+    this.creating = true;
+    this._notificationService.success(`Created the ${cluster.name} cluster`);
+    this._googleAnalyticsService.emitEvent('clusterCreation', 'clusterCreated');
```

Comment:
Why is it there? We have removed most of the GA usages as it is not configured anyway and not properly used.